zulipterminal/ui.py

```diff
@@ -228,19 +258,19 @@ def keypress(self, size: urwid_Box, key: str) -> Optional[str]:
             self.body.focus_col = 1
         elif is_command_key("SEARCH_PEOPLE", key):
             # Start User Search if not in editor_mode
-            self.body.focus_position = 2
-            self.users_view.keypress(size, key)
             self.show_left_panel(visible=False)
             self.show_right_panel(visible=True)
+            self.body.focus_position = 2
+            self.users_view.keypress(size, key)
```

Comment:
Yes, it's because of the override but this was not the one we removed in one of the previous refactor commits. That was focus_setting when visible = True this is when visible = False